src/integration-test/java/com.accounts.integration.tests/AccountsApplicationRunner.java

```diff
@@ -0,0 +1,17 @@
+package com.accounts.integration.tests;
+
+import com.accounts.app.Application;
+import io.restassured.RestAssured;
+import org.testng.annotations.BeforeClass;
+
+public class AccountsApplicationRunner {
+
+    @BeforeClass
+    public void startApp() throws InterruptedException {
+        new Application().start();
+        //TODO waiting for app
+        Thread.sleep(1000);
```

Comment:
Shouldn't we wait for the event instead of making the thread sleep

Reply:
Yes, that's why there is todo comment